client/src/handlebars/helpers.side-effects.js

```diff
@@ -459,7 +427,7 @@ Handlebars.registerHelper("le_dept",function(context){
   if (article.length > 0 && _.last(article) !== "'") {
     article += " ";
   }
-  return article + (dept.applied_title || dept.name);
+  return article + (dept.name);
```

Comment:
I suppose this could be written as string interpolation but such a minor pick since it's just simple string concatenation

Reply:
I think I didn't bother cleaning it up more since the handelbars helpers file is such old junky code. Broken windows and all that.